docs/known_issues.md

```diff
@@ -19,6 +19,15 @@ Install `baybe` into a clean environment without pre-existing
 packages. If you require other packages, try to install `baybe` first.
 ```
 
+### CPUs without AVX support - Installation of `polars`
+The package `polars` that can be installed as an optional dependency is only supported for
+CPUs with AVX support. As a consequence, you might not be able to install the optional dependency.
+This is in particular the case for M1 Macs, as these do not offer this support.
+
+If you want to use the optional `polars` dependency on such a system, you need to install
+`polars-lts-cpu` instead, preferably via `pip`. For more details, we refer to the
+[polars installation guide](https://docs.pola.rs/user-guide/installation/).
```

Comment:
```suggestion
```{admonition} Suggested Fix
:class: tip
Instead of `polars`, install `polars-lts-cpu`. BayBE will automtatically detenct the presence of `polars` and active its advanced machinery. For more details, we refer to the
[polars installation guide](https://docs.pola.rs/user-guide/installation/).
```
```

Reply:
formatting seems to be messed u on github

Reply:
Committed your suggestion, but what do you mean with your second command? It seems like the formatting of this file is in general messed up on github, is this what you mean?

Reply:
no look at the bottom here on github its messed up (shouldnt affect the file)

Reply:
Do you mean this?
![image](https://github.com/user-attachments/assets/78631765-1fd0-4c71-aaf1-ec2c8dbc5e88)
This is just github not being able to parse the admonition. Or what do you mean? Can you share a screenshot?

Reply:
i meant this here at the bottom 
![image](https://github.com/user-attachments/assets/a2644421-7227-4f59-9ae3-608b452f4f05)

Reply:
Ok, I think I get what you mean. I slightly changed your suggestion anyway (fixed some typos and some backticks were missing), so I think everything should be gucci now?